seed/csharp-sdk/auth-environment-variables/src/SeedApi.DynamicSnippets/Example0.cs

```diff
@@ -0,0 +1,20 @@
+using global::System.Threading.Tasks;
+using SeedAuthEnvironmentVariables;
+using SeedAuthEnvironmentVariables.Core;
+
+namespace Usage;
+
+public class Example0
+{
+    public async global::System.Threading.Tasks.Task Do() {
+        var client = new SeedAuthEnvironmentVariablesClient(
+            apiKey: "<value>",
+            clientOptions: new ClientOptions{
+                BaseUrl = "https://api.fern.com"
+            }
+        );
+
+        await client.Service.GetWithApiKeyAsync();
+    }
+
```

Comment:
Extra newline not necessary here

Reply:
Noted - I left this change out for now since I know you're editing some of the `Method` AST node in your change. I'll come back to this. Plus, this representation isn't actually shown to the user - the top-level statements form will be.